package.json

```diff
@@ -44,7 +44,7 @@
     "@carbon/icons-react": "^11.20.0",
```

Comment:
You can go through https://github.com/carbon-design-system/carbon/pull/13932/files and do a search for each of the `@carbon` packages listed here, and update the version to that `rc`